ci-infra/azure/main.tf

```diff
@@ -72,6 +72,16 @@ resource "azurerm_shared_image_gallery" "podvm_image_gallery" {
   name                = "${var.image_gallery}${var.ver}"
   resource_group_name = azurerm_resource_group.ci_rg.name
   location            = azurerm_resource_group.ci_rg.location
+
+  sharing {
+    permission = "Community"
+    community_gallery {
+	prefix = "cocopodvm"
+	eula   = "https://raw.githubusercontent.com/confidential-containers/confidential-containers/main/LICENSE"
+	publisher_uri = "https://github.com/confidential-containers/confidential-containers"
+	publisher_email = "kartikjoshi@microsoft.com"
```

Comment:
We need some generic email id isn't it?

Reply:
Yess it should be something generic, i saw other community gallery there we are using Magnus's id so for time being i used this one. What do you suggest which one we should use?

Reply:
We can use the coco email id: cocoatmsft@outlook.com

Reply:
or if @bpradipt has some suggestions on this one?

Reply:
[cocoatmsft@outlook.com](mailto:cocoatmsft@outlook.com) looks fine.